sample/DokanNetMirror/Mirror.cs

```diff
@@ -380,7 +380,18 @@ public NtStatus SetFileAttributes(string fileName, FileAttributes attributes, Do
         {
             try
             {
+                var stream = info.Context as FileStream;
+                if (stream != null)
+                {
+                    var ct = creationTime?.ToFileTime() ?? 0;
+                    var lat = lastAccessTime?.ToFileTime() ?? 0;
+                    var lwt = lastWriteTime?.ToFileTime() ?? 0;
+                    if (Win32.SetFileTime(stream.SafeFileHandle, ref ct, ref lat, ref lwt)) return NtStatus.Success;

```

Comment:
Please move the return to a new line.

You can have different tastes, but I would have preferred if it looked like this:
```C#
if (!Win32.SetFileTime(stream.SafeFileHandle, ref ct, ref lat, ref lwt))
{
    throw Marshal.GetExceptionForHR(Marshal.GetLastWin32Error());
}
return NtStatus.Success;

```